src/stores/widgetManager.ts

```diff
@@ -443,7 +443,8 @@ export const useWidgetManagerStore = defineStore('widget-manager', () => {
 
   // If the user does not have it's own profiles yet, try to fetch them from the vehicle, and if it fails, create default ones
   if (savedProfiles.value.isEmpty()) {
-    importProfilesFromVehicle()
+    // We use a self invoked await function to avoid moving the entire store to async
+    ;(async () => importProfilesFromVehicle())()
```

Comment:
I'm not familiar with that workaround. Will this actually wait?

Reply:
It's explained here: https://stackoverflow.com/questions/40745153/es8-immediately-invoked-async-function-expression

Reply:
Interesting. More on that [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/async_function).